### PR TITLE
Added code in pyscan to send light and accelerometer data to pybytes

### PR DESCRIPTION
**Pull request by Aditya18Jain**

### CODE REVIEW

**Comment by Xykon** [comment]
These are general libraries they should not contain any Pybytes code.

We consider Pybytes an optional platform for customers to use while these libraries are for general purpose. If anything you can add a Pybytes example in a separate main.py file.

Otherwise nice work adding the missing libraries. I'm assigning this to Daniel S. for final review/merge as I'm in China and busy with other stuff plus might also need to update the docs accordingly.

File: pyscan/main.py [hunk 1]
```diff
@@ -42,6 +46,12 @@ def print_debug(msg):
     if DEBUG:
         print(msg)
 
+def send_sensor_data(name, timeout):
+    while(pybytes):
+        pybytes.send_virtual_pin_value(True, 2, lt.light())
+        pybytes.send_virtual_pin_value(True, 3, li.acceleration())
```
Comment (sdaniel55) [inline pyscan/main.py]: Hi @Aditya18Jain 👋 

Could you change these to simple `print()`? That's what we use in the other examples as well. By default, these examples should run on every devices/firmware type. I'll merge this after that's changed. Thanks!

Comment (Aditya18Jain) [reply pyscan/main.py]: replaced the code with suggested changes.

File: pyscan/main.py [hunk 2]
```diff
@@ -42,6 +46,12 @@ def print_debug(msg):
     if DEBUG:
         print(msg)
 
+def send_sensor_data(name, timeout):
+    while(pybytes):
```
Comment (sdaniel55) [inline pyscan/main.py]: @Aditya18Jain I think this line is not needed either, since it's not using Pybytes. Sorry, I missed this before

**Pull request merged**